### PR TITLE
fix: skip tags setup

### PR DESCRIPTION
**Pull request by denis-ev**
Signed-off-by: Denis Evers <denis-ev@users.noreply.github.com>

### CODE REVIEW

**Comment by denis-ev** [comment]
fixes #315

File: tasks/utils.rb [hunk 1]
```diff
@@ -8,42 +8,33 @@ module Utils
   include ConfigFileUtils
   include VlabI18n
 
-  def run_playbook(playbook, options, extra = nil)
+  def run_playbook(playbook, options, extra = nil, deploy: false)
```
Comment (codefriar) [inline tasks/utils.rb:11]: if you say `deploy: false` it will always be false. I suspect you want deploy = false ?

File: tasks/utils.rb [hunk 2]
```diff
@@ -8,42 +8,33 @@ module Utils
...
+    command << '--skip-tags bastion' unless decrypted_config_file[:bastion][:enable] and not deploy rescue false
     command << "-e config_dir=#{options[:config_dir]}"
     command << extra.to_s unless extra.nil? || extra.size.zero?
     command << '-i inventory'
     command.join(' ')
   end
 
-  def run_config_playbook(options, extra = '')
-    playbook_cmd = playbook_command('playbook.config.yml', extra, options[:debug].to_sym).strip
+  def run_config_playbook(options, extra = '', deploy: false)
```
Comment (codefriar) [inline tasks/utils.rb:36]: you need deploy = false

File: tasks/utils.rb (hunk 1 above)
Comment (denis-ev) [reply tasks/utils.rb:11]: https://docs.rubocop.org/rubocop/cops_style.html#styleoptionalbooleanparameter

**Review by codefriar — APPROVED** [review]

**Pull request merged**